.wharf-ci.yml

```diff
@@ -1,10 +1,10 @@
 build:
-  api:
+  wharf-api:
     docker:
       file: Dockerfile
-      tag: ${GIT_COMMIT},${GIT_TAG},latest
-      append-cert: false
+      tag: canary
       args:
-        - BUILD_VERSION=${GIT_TAG}
+        - BUILD_VERSION=${GIT_BRANCH}
```

Comment:
Should this really be changed?

Reply:
The idea is not to build released versions, but instead any arbitrary commit from any branch. So the `GIT_TAG` Wharf variable will be empty most of the times, and the `BUILD_VERSION` Docker argument is set to `GIT_BRANCH` mostly just to get some kind of info about where the build is from.

For example, I used these builds and deployed them to our internal staging env, resulting in:

![Screenshot from 2021-11-24 09-16-19](https://user-images.githubusercontent.com/2477952/143200374-b34c7cf6-e018-4f1a-bf2f-9c0abadded5d.png)

In contrast to:

![Screenshot from 2021-11-24 09-16-57](https://user-images.githubusercontent.com/2477952/143200420-edb16bec-936a-42e7-a85d-9cd5a478f11e.png)